bridge/contracts/InvalidTxConsumptionAccusation.sol

```diff
@@ -34,72 +35,89 @@ contract InvalidTxConsumptionAccusation is
-    /// @param _proofs an array of merkle proof structs in the following order:
-    /// proof against StateRoot: Proof of inclusion or exclusion of the deposit or UTXO in the stateTrie
-    /// proof of inclusion in TXRoot: Proof of inclusion of the transaction that included the invalid input in the txRoot trie.
-    /// proof of inclusion in TXHash: Proof of inclusion of the invalid input (txIn) in the txHash trie (transaction tested against the TxRoot).
-    /// @return the address of the signer
+    /**
+     * @notice This function validates an accusation of non-existent utxo consumption, as well as invalid deposit consumption.
+     * @param pClaims_ the PClaims of the accusation
+     * @param pClaimsSig_ the signature of PClaims
+     * @param bClaims_ the BClaims of the accusation
+     * @param bClaimsSigGroup_ the signature group of BClaims
+     * @param txInPreImage_ the TXInPreImage consuming the invalid transaction
+     * @param proofs_ an array of merkle proof structs in the following order:
+     * proof against StateRoot: Proof of inclusion or exclusion of the deposit or UTXO in the stateTrie
+     * proof of inclusion in TXRoot: Proof of inclusion of the transaction that included the invalid input in the txRoot trie.
+     * proof of inclusion in TXHash: Proof of inclusion of the invalid input (txIn) in the txHash trie (transaction tested against the TxRoot).
+     * @return the address of the signer
+     */
     function accuseInvalidTransactionConsumption(
-        bytes memory _pClaims,
-        bytes memory _pClaimsSig,
-        bytes memory _bClaims,
-        bytes memory _bClaimsSigGroup,
-        bytes memory _txInPreImage,
-        bytes[3] memory _proofs
+        bytes memory pClaims_,
+        bytes memory pClaimsSig_,
+        bytes memory bClaims_,
+        bytes memory bClaimsSigGroup_,
+        bytes memory txInPreImage_,
+        bytes[3] memory proofs_
     ) public view returns (address) {
         // Require that the previous block is signed by correct group key for validator set.
-        _verifySignatureGroup(_bClaims, _bClaimsSigGroup);
+        _verifySignatureGroup(bClaims_, bClaimsSigGroup_);
 
         // Require that height delta is 1.
-        BClaimsParserLibrary.BClaims memory bClaims = BClaimsParserLibrary.extractBClaims(_bClaims);
-        PClaimsParserLibrary.PClaims memory pClaims = PClaimsParserLibrary.extractPClaims(_pClaims);
+        BClaimsParserLibrary.BClaims memory bClaims = BClaimsParserLibrary.extractBClaims(bClaims_);
+        PClaimsParserLibrary.PClaims memory pClaims = PClaimsParserLibrary.extractPClaims(pClaims_);
 
         if (pClaims.bClaims.txCount == 0) {
             revert AccusationsErrors.NoTransactionInAccusedProposal();
         }
 
-        // todo: check if the height is at least 2 epochs behind
+        Snapshot memory latestSnapshot = ISnapshots(_snapshotsAddress()).getLatestSnapshot();
```

Comment:
Since these 2 new variables are needed after line 76, I suggest moving these 2 lines down to line 76.

Reply:
I agree!